app/mailers/devise_custom_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Mailer Overrides for Devise default emails
+class DeviseCustomMailer < Devise::Mailer
+  include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
+  default template_path: 'devise/mailer' # to make sure that your mailer uses the devise views
+
+  def confirmation_instructions(record, token, opts = {})
+    @greeting_name = record.greeting_name
+    @token = token
+    opts[:subject] = 'Pie for Providers email verification'
+    attachments.inline['pieFullTanLogo.svg'] = File.read(Rails.root.join('app/views/devise/mailer/assets/pieFullTanLogo.svg'))
```

Comment:
why this path?  why isn't the `.svg` under the conventional `app/assets/images...` path?

Reply:
Because API-only Rails doesn't load assets through the asset pipeline, so it doesn't really matter where it's stored, and I thought it made more sense to put it near the mailers where it's used rather than an asset pipeline folder that isn't being used as it would be in a non-API implementation